libparsec/crates/platform_mountpoint/src/unix/mod.rs

```diff
@@ -0,0 +1,436 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+use fuser::{
+    FileAttr, FileType, Filesystem, MountOption, ReplyAttr, ReplyCreate, ReplyData, ReplyDirectory,
+    ReplyEmpty, ReplyEntry, ReplyOpen, ReplyWrite, Request, Session, SessionUnmounter,
+};
+use libc::{ENAMETOOLONG, ENOENT};
+use libparsec_types::{anyhow, FileDescriptor};
+use std::{
+    ffi::OsStr,
+    path::{Path, PathBuf},
+    time::Duration,
+};
+
+use crate::{
+    error::MountpointError, EntryInfo, EntryInfoType, MountpointInterface, MountpointManager,
+    WriteMode,
+};
+
+#[derive(Debug, Clone, Copy)]
+pub(crate) struct Counter(pub u64);
+
+impl Counter {
```

Comment:
Add doc comments to explain why this counter exists.

Reply:
Done